vtex/profile-form
# Add new it locale translations

#### What is the purpose of this pull request?
To change some `it` locale translations

#### What problem is this solving?
`miriadeit` is an italian store and they need the proper translation for italian strings

#### How should this be manually tested?
- Enter `phdro--miriadeit.myvtex.com`
- Log in to My Account to see the form with the new strings

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/19555647/82510570-4427ce80-9ae1-11ea-874d-758b6bf240d1.png)

![image](https://user-images.githubusercontent.com/19555647/82510533-29edf080-9ae1-11ea-83d8-0f45d9b2d9f1.png)


#### Types of changes
